.github/workflows/linux.yml

```diff
@@ -162,11 +162,11 @@ jobs:
     - uses: actions/checkout@v4
     - name: setup apt repo
       run: |
-        # oneapi-ci/scripts/setup_apt_repo_linux.sh
-        wget https://apt.repos.intel.com/intel-gpg-keys/GPG-PUB-KEY-INTEL-SW-PRODUCTS-2023.PUB
```

Comment:
Just out of curiosity: Would it be enough to switch to `https://apt.repos.intel.com/intel-gpg-keys/GPG-PUB-KEY-INTEL-SW-PRODUCTS.PUB`? I'll try that out on my fork.

Reply:
Was able to fix my oneAPI woes using this method. Many whanks!
Using WSL2 Ubuntu